src/render.spec.ts

```diff
@@ -1,103 +0,0 @@
-import expect from "expect";
```

Comment:
Sad to see 1 day of work being trashed... But that's what we do! This may help you to understand how much time to spend on a feature.